### PR TITLE
docs: mention GitHub support for Mermaid

### PR DESCRIPTION
**Pull request by kdambekalns**

### CODE REVIEW

File: docs/pages/mermaid.md [hunk 1]
```diff
@@ -24,8 +24,6 @@ diagrams dynamically*. The syntax is easy and clear, and the output usually beau
 
 See [mermaid-js#integrations](https://mermaid-js.github.io/mermaid/#/integrations?id=integrations) for a full list.
 
-**GitHub** does not support it for now, but there are many browser extensions available
-that will automatically detect and render the diagrams in md files.
-I personally use [mermaid-extension](https://github.com/Stefan-S/mermaid-extension).
+**GitHub** does support it, too
 
-[Enable Mermaid rendering in GitHub READMEs](https://mermaid.stefs.me/){: .btn .btn-outline }
+[Creating Mermaid diagrams](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-diagrams#creating-mermaid-diagrams){: .btn .btn-outline }
```
Comment (derlin) [inline docs/pages/mermaid.md]: May I suggest adding GitHub to the list (line 22) instead and removing completely this sentence instead?

Something like:
```
* [GitHub](https://github.com), see their documentation [Creating Mermaid diagrams](https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-diagrams#creating-mermaid-diagrams)
```

**Comment by derlin** [comment]
Thank you for the contribution, that's awesome!

I completely missed this update from GitHub, very good call. Before going further, this repository uses [conventional commits](conventionalcommits.org/en/v1.0.0/) and a rebase+squash merge strategy. 

Can you please ensure the final PR contains only one commit and has a commit message following conventional commits convention? Something like:

```
docs: mention GitHub support

<the description you want>
```

Again, thank you for taking the time, and welcome :)

**Comment by kdambekalns** [comment]
> has a commit message following conventional commits convention?

Dang, I looked at the commit history to find out what's commonly used here. Shouldn't have scrolled down… 🙃

**Review by derlin — APPROVED** [review]

**Comment by derlin** [comment]
Great, thanks!

**Pull request merged**